pkg/resourceutils/scaleupduringdr.go

```diff
@@ -0,0 +1,347 @@
+package resourceutils
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	migration "github.com/libopenstorage/stork/pkg/migration/controllers"
+	"github.com/libopenstorage/stork/pkg/resourcecollector"
+	"github.com/portworx/sched-ops/k8s/core"
+	storkops "github.com/portworx/sched-ops/k8s/stork"
+	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	k8sdynamic "k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+)
+
+func ScaleUpResourcesInNamespace(namespace string, dryRun bool, config *rest.Config) (map[metav1.GroupVersionKind]map[string]string, error) {
```

Comment:
I have moved the functionality of GetResourcesBeingActivated from listResources file here.
Have refactored the code a bit and only added the actual update of the resource compared to previous GetResourcesBeingActivated method which was only creating the map.